src/doom/hu_stuff.c

```diff
@@ -217,11 +217,7 @@ char*   mapnames_chex[] =   // Chex Quest names.
     "NEWLEVEL"
 };
 
-// List of names for levels in commercial IWADs
```

Comment:
Please see commit b777eebd4d4bd3b974b11e1fc4968e6f2b0e65d5 - these map names are in a single array for a reason. I'm a bit concerned your change probably breaks some of the intended overflow behavior.